docs/src/trace-viewer.md

```diff
@@ -160,6 +160,13 @@ Available options to record a trace:
 
 You can also use `trace: 'retain-on-failure'` if you do not enable retries but still want traces for failed tests.
 
+To save disk space, you can use an object to configure the trace options and disable features like screenshots:
```

Comment:
Perhaps we should just link to https://playwright.dev/docs/api/class-testoptions#test-options-trace? That would be:

```
There are more granular options available, see [`property: TestOptions.trace`].
```

Let me know what you think.

Reply:
Thanks I like your suggestion to link to the docs much more. So there is no need to update the docs in two places if the TestOptions.trace changes. 👍 
Because I took so long, I've opened a new PR: https://github.com/microsoft/playwright/pull/31182